src/molecules/side-menu.module.css

```
@@ -0,0 +1,92 @@
+.sideMenu {
+    position: fixed;
```

```suggestion
    position: fixed;
    z-index: 10;
```
z 位置を指定しないと, 後ろにあるテキストなどが透けて出てきてしまいます